package.json

```diff
@@ -0,0 +1,5 @@
+{
+  "dependencies": {
+    "eslint": "^8.25.0"
```

Comment:
I think this file shouldn't be here either

Reply:
will delete it.

Reply:
you should also remove `package-lock.json`